### ISSUE TITLE
Error 'CLIPTextTransformer' object has no attribute '_build_causal_attention_mask'

### ISSUE THREAD

**Issue by tudal** [andersxa/comfyui-PromptAttention#2]
I'm getting this error:
![image](https://github.com/andersxa/comfyui-PromptAttention/assets/799063/b80192e0-6585-436e-87c9-3ce3e67b2ba6)

**Comment by skywalkerx86** [comment]
also get this error. I'm not familiar enough with the underlying system but looking at the code and the source for the clip transformers, the class  CLIPTextTransform (defined in modeling_clip.py) builds it's causal attention mask from  a function _make_causal_mask that is not defined within itself (not part of the class) whereas the _build_causal_attention_mask method seems to be defined in the TFClipTextTransform class (defined in modeling_tf_clip.py).

**Comment by PaulFidika** [comment]
Same
